### ISSUE TITLE
Xcomfort X-1 controller?

### ISSUE THREAD

**Issue by solano9** [hteibler/eaton_docker#2]
Should this also work with the xcomfort X-1 controller?

**Comment by hteibler** [comment]
Not sure which device you mean?
this is for:
https://www.eaton.com/at/de-de/catalog/residential/xcomfort-controllers.html

**Comment by solano9** [comment]
As I understand, your solution is made (only) for xcomfort SHC CHCA-00/01.
There is also a controller named Sensio X-1 (https://shop.sensio.no/produkt/sensio-gateway/) that seems to use the same hardware, but with Sensio software with some unknown API. I tried your solution today in docker towards my Sensio X-1 controller, but I get "Connection refused":

My API-request (I have replaced my actual ip with "dockerip"):
http://dockerip:9997/actor?actor=xCo:5951286_u0&set=dim:80
returns
```
{
    "status": "error",
    "error": "HTTPConnectionPool(host='dockerip', port=80): Max retries exceeded with url: /remote/json-rpc (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f945437a0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
}
```

I am not sure if it is even possible to get info on the API for my controller and make it work in a similar fashion, but it would be interresting if you would share some thougts based on your experience working with the Xcomfort SHC.

**Comment by hteibler** [comment]
have you adjusted the "parameter.py" file?

**Comment by solano9** [comment]
Yes, I did that before building the image and starting the container with compose.
